Products/ZenHub/tests/test_PBDaemon.py

```diff
@@ -1591,14 +1590,33 @@ def test__signalZenHubAnswering_False(t, zenPath, os):
         zenPath.assert_called_with('var', filename)
         os.remove.assert_called_with(zenPath.return_value)
 
-    @patch('{src}.publisher'.format(**PATH), autospec=True)
-    def test_buildOptions(t, publisher):
+    from unittest import skip
```

Comment:
Shouldn't this go to the top of the module?

Reply:
fixed